pkg/controllers/generic/patcher.go

```diff
@@ -56,7 +57,8 @@ func (s *patcher) ApplyPatches(ctx context.Context, fromObj, toObj client.Object
 		// always apply status if it's there
 		if hasAfterStatus {
 			s.log.Infof("Apply status of %s during patching", toObjCopied.GetName())
-			err = s.toClient.Status().Patch(ctx, toObjCopied.DeepCopy(), client.Apply, client.ForceOwnership, client.FieldOwner(fieldManager))
+			o := &client.SubResourcePatchOptions{PatchOptions: client.PatchOptions{FieldManager: fieldManager, Force: pointer.Bool(true)}}
```

Comment:
Based on:
https://github.com/kubernetes-sigs/controller-runtime/blob/4a538741e9aea74df6caf4670b05520b8adb6ea2/pkg/client/options.go#L786-L789
and
https://github.com/kubernetes-sigs/controller-runtime/blob/4a538741e9aea74df6caf4670b05520b8adb6ea2/pkg/client/options.go#L143-L145

Reply:
Yes. I understood this one now.